sdks/python/apache_beam/examples/fastavro_it_test.py

```diff
@@ -124,23 +123,22 @@ def batch_indices(start):
         | 'create-records' >> Map(record)
 
     fastavro_output = '/'.join([self.output, 'fastavro'])
-    avro_output = '/'.join([self.output, 'avro'])
+    # avro_output = '/'.join([self.output, 'avro'])
 
     # pylint: disable=expression-not-assigned
     records_pcoll \
     | 'write_fastavro' >> WriteToAvro(
         fastavro_output,
         parse_schema(json.loads(self.SCHEMA_STRING)),
-        use_fastavro=True
     )
 
-    # pylint: disable=expression-not-assigned
-    records_pcoll \
-    | 'write_avro' >> WriteToAvro(
-        avro_output,
-        Parse(self.SCHEMA_STRING),
-        use_fastavro=False
-    )
+    # # pylint: disable=expression-not-assigned
```

Comment:
We do not keep commented-out code - we can just remove this.
If we ever need this code again, we can look it up in revision history.